docs/users-guide/models.md

```diff
@@ -0,0 +1,107 @@
+
+Models:
+
+- Let you update column descriptions and customize metadata to create great starting points for exploration.
+- Show up higher in search results and get highlighted when other users start new questions to promote reuse.
+- Live in collections to keep them separate from messy database schemas.
+
+## How to use models
+
+You can use models to:
+
+- Create, uh, models, with model here meaning a simplified description of some concept in your business that you codify as a set of columns. An example model could be a "customer", which is a table that pulls together customer information from multiple tables and adds computed columns, like adding a lifetime value (LTV) column. This model represents the [measures and dimensions][measures-dimensions] that you think are relevant to your understanding of your customers.
+- Create summary tables that aggregate data from multiple tables.
+- Clean up tables with unnecessary columns and rows filtered out.
+
+The idea with models is to give other people a good "starting point table" that makes it easier to answer any questions they have about the subject being modeled.
+
+## Create a model
+
+First, search for models that already exist. If you can't find one that meets your needs, you can create a model like so:
+
+1. [Ask a question][question] using either the query builder or the SQL editor.
+2. Save the question.
+3. Click on the down arrow next the question title to open the question details sidebar.
+4. Click on the model icon to turn the question into a model.
+
+## Add metadata to columns in a model
+
+Metadata is the secret sauce of models. When you write a SQL query, Metabase can display the results, but it can't "know" what kind of data it's returning (like it can with questions built using the query builder). What this means in practice is that people won't be able to drill-through the results, because Metabase doesn't understand what the results are. With models, however, you can tell Metabase what kind of data is in each returned column so that Metabase can still do its drill-through magic. If you only set one kind of metadata, set the **Column type** to let Metabase know what kind of data it's working with.
+
+### Display name
+
+What people will see as the column's name.
+
+### Description
+
+A place to write helpful context for the column.
+
+### Column type
+
+You can set the [column type][column-type]. The default is "No special type". Be sure to set the Column type so people can interact with the results.
+
+## This column should appear in...
+
+- Table and detail views
+- Detail views only
+
+### Column settings
+
+You can specify whether a column should appear in the table view, or just in a detail view (when you click on the entity/primary key for the row).
+
+### Display as
```

Comment:
Other options appear here if the column is an image or an email address I think. You can cross-reference this setting from admin/datamodel, as it's the same setting.